parlai/scripts/train_model.py

```diff
@@ -182,7 +182,16 @@ def setup_args(parser=None) -> ParlaiParser:
         'ppl,f1,accuracy,hits@1,rouge,bleu'
         'the rouge metrics will be computed as rouge-1, rouge-2 and rouge-l',
     )
+    train.add_argument(
```

Comment:
perhaps a totally different issue here but seeing this param added twice leads me to believe we should have some sort of `common_script_args` that we can add for all our scripts (would help standardize command line params e.g. how `-n` is num examples in some while `-ne` is num examples in others)